lib/pharos/phases/configure_client.rb

```diff
@@ -18,8 +18,6 @@ def call
         return if @optional && !kubeconfig?
 
         cluster_context['kubeconfig'] = kubeconfig
-        cluster_context['master-ssh'] = ssh
```

Comment:
Unnecessary, the phase.rb can whip up an ssh client to a host with `master_sort_score.zero?`.